include/bluetooth/mesh/sensor_srv.h

```diff
@@ -58,6 +58,38 @@ struct bt_mesh_sensor_srv;
 					      _srv),                           \
 			 &_bt_mesh_sensor_setup_srv_cb)
 
+#ifdef CONFIG_BT_MESH_LARGE_COMP_DATA_SRV
+/** @def BT_MESH_MODEL_SENSOR_SRV_WITH_METADATA
+ *
+ *  @brief Sensor Server model composition data entry with metadata.
+ *
+ *  @param[in] _srv Pointer to a @ref bt_mesh_sensor_srv instance.
+ *  @param[in] _metadata Pointer to a @ref bt_mesh_models_metadata_entry.
+ */
+#define BT_MESH_MODEL_SENSOR_SRV_WITH_METADATA(_srv, _metadata)                                    \
+	BT_MESH_MODEL_METADATA_CB(BT_MESH_MODEL_ID_SENSOR_SRV, _bt_mesh_sensor_srv_op,             \
+				  &(_srv)->pub,                                                    \
+				  BT_MESH_MODEL_USER_DATA(struct bt_mesh_sensor_srv, _srv),        \
+				  &_bt_mesh_sensor_srv_cb, &_metadata),                     \
+	BT_MESH_MODEL_CB(BT_MESH_MODEL_ID_SENSOR_SETUP_SRV, _bt_mesh_sensor_setup_srv_op,          \
+			 &(_srv)->setup_pub,                                                       \
+			 BT_MESH_MODEL_USER_DATA(struct bt_mesh_sensor_srv, _srv),                 \
+			 &_bt_mesh_sensor_setup_srv_cb)
+#endif
+
+/** Sensor Properties Metadata ID. */
+#define BT_MESH_SENSOR_PROP_METADATA_ID 0x0001
+
+/**
+ *  Sensor Properties Metadata entry.
+ *
+ *  @param[in] ... list of 16-bit sensor property IDs split by comma.
+ */
+#define BT_MESH_SENSOR_PROP_METADATA(...)                                                          \
+	BT_MESH_MODELS_METADATA_ENTRY(2 * ARRAY_SIZE(((uint16_t[]){__VA_ARGS__})),                 \
```

Comment:
Could probably be simply `sizeof` to reduce mathematics:
```suggestion
	BT_MESH_MODELS_METADATA_ENTRY(sizeof(((uint16_t[]){__VA_ARGS__})),                 \
```

Reply:
this is preprocessor mathematic that doesn't impact on anything but Ok